setup.py

```diff
@@ -35,6 +36,14 @@
         opt_reqs += [ir]
 extras_require['extras'] = opt_reqs
 
+# If interested in benchmarking devito, we need the `examples` too
+exclude = ['docs', 'tests']
+try:
+    if not bool(int(os.environ.get('DEVITO_BENCHMARKS', 0))):
+        exclude += ['examples']
+except (TypeError, ValueError):
+    exclude += ['examples']
```

Comment:
That's pretty :D

Reply:
yeah, makes puke right ? 😬 

I thought I had linked u this morning. Of course, if you have better suggestions, I'm all ears!

Reply:
Don't really have a better one except maybe `argparser` but not much better